scripts/script-free.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Check if Account ID and Delegate Token is provided as an argument
+if [ $# -ne 3 ]; then
+    echo "Usage: $0 <ACCOUNT_ID> <DELEGATE_TOKEN> <DELEGATE_IMAGE_VERSION>"
+    exit 1
+fi
+# Get the operating system type
+ostype=$(uname)
+# Get the machine architecture type
+archtype=$(uname -m)
+
+if [ "$archtype" = "x86_64" ]; then
+    archtype="amd64"
+fi
+
+# Account ID from script argument
+ACCOUNT_ID=$1
+# Account ID from script argument
+DELEGATE_TOKEN=$2
+DELEGATE_IMAGE_VERSION=$3
+# Check the operating system type and run commands accordingly
+if [ "$ostype" = "Darwin" ]; then
+    # Commands for macOS
+    echo "Running commands for macOS"
+
+    # Run the Docker container with environment variables
+    docker run -d --cpus=1 --memory=2g --restart=always\
+      -e DELEGATE_NAME=docker-delegate \
+      -e NEXT_GEN="true" \
+      -e DELEGATE_TYPE="DOCKER" \
+      -e ACCOUNT_ID=$ACCOUNT_ID \
+      -e DELEGATE_TOKEN=$DELEGATE_TOKEN \
+      -e LOG_STREAMING_SERVICE_URL=https://app.harness.io/gratis/log-service/ \
```

Comment:
Free user logins are created on prod 1. Will the values be the same?

Reply:
"free" here comes from the env value, which is prod2. WIll create a seperate script for prod1.